app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityController.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.player.exploration
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import javax.inject.Inject
+
+/** The controller for [ExplorationActivity]. */
+@ActivityScope
+class ExplorationActivityController @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.exploration_activity)
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.exploration_fragment_placeholder,
+      ExplorationFragment()
```

Comment:
I noticed a mistake with this pattern in #106: we need to check whether the fragment is already inflated to avoid re-adding it upon configuration changes. See HomeActivity in that PR for how I did this (tl;dr is I added a helper to retrieve the fragment and used that to check presence before reinflation).

Reply:
Added `getExplorationFragment()` and add fragment only if it is not present.

Solved as per comment.